README.md

```diff
@@ -4,34 +4,13 @@
 [![devDependency Status](https://david-dm.org/ng-bootstrap/core/dev-status.svg?branch=master)](https://david-dm.org/ng-bootstrap/core#info=devDependencies)
 
 Welcome to the Angular 2 version of the [Angular UI Bootstrap](https://github.com/angular-ui/bootstrap) library.
-This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework.
+This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework. 
+This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
```

Comment:
Yea, we can combine the two like:  "Welcome to the Angular 2 version of the ... library built by the ... team." or something similar.